src/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -212,10 +212,11 @@ protected virtual AuthenticationResult PostTokenRequest()
 
         private async Task SendHttpMessageAsync(OAuth2Client client)
         {
+            UriBuilder builder = new UriBuilder(AuthenticationRequestParameters.Authority.TokenEndpoint);
+            builder.AppendQueryParameters("slice=testslice&uid=true");
```

Comment:
can you file an issue to make sure that we need to remove slice=testsclice once the feature is in production?